tests/test_models.py

```diff
@@ -139,6 +147,9 @@ def test_question_text_filter(tmp_path):
       </questions>
     </question_versions>
   </question_category>
+        </question_bank_entry>
+      </question_bank_entries>
+
```

Comment:
Let's update this in-line fixture to be a bit more correct in terms of representing the XML structure in `questions.xml` + indented consistently with others:

```python
question_bank_xml = """
<?xml version="1.0" encoding="UTF-8"?>
<question_categories>
  <question_category>
    <question_bank_entries>
      <question_bank_entry id="1">
        <idnumber>1234</idnumber>
        <question_version>
          <question_versions>
            <version>1</version>
            <questions>
              <question id="1">
                <qtype>shortanswer</qtype>
                <questiontext>{1:SHORTANSWER:%100%2#Congratulations!}</questiontext>
              </question>
            </questions>
          </question_versions>
        </question_version>
      </question_bank_entry>
    </question_bank_entries>
  </question_category>
</question_categories>
    """.strip()
```